timesketch/lib/utils.py

```diff
@@ -73,6 +73,18 @@ def _parse_tag_field(row):
     return [row]
 
 
+def _scrub_special_tags(dict_obj):
+    """Remove Elastic specific fields from a dict."""
+    if '_id' in dict_obj:
```

Comment:
Maybe something like:

```
fields_to_remove = ['_id', '_type', '_index', '_source']
for field in fields_to_remove:
    if field in dict_obj:
        _ = dict_obj.pop(field)
```

Reply:
sure, can do that (this was copy / paste) yours is a bit cleaner ;)